### PR TITLE
Add beanstalkd_exporter to the list of exporters

### PR DESCRIPTION
**Pull request by marcelcorso**
An exporter of [beanstalkd](http://kr.github.io/beanstalkd) stats

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [RabbitMQ exporter](https://github.com/kbudde/rabbitmq_exporter)
    * [RabbitMQ Management Plugin exporter](https://github.com/deadtrickster/prometheus_rabbitmq_exporter)
    * [Mirth Connect exporter](https://github.com/vynca/mirth_exporter)
+   * [Beanstalkd_exporter](https://github.com/messagebird/beanstalkd_exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Keep it sorted

**Comment by brian-brazil** [comment]
Please address the comment on this PR so it can be merged.

**Comment by marcelcorso** [comment]
@samwierema is working on this and soon we'll give an update

**Comment by brian-brazil** [comment]
I'm still waiting on the new item to be sorted in the list.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**